packages/blocks/src/page-block/utils/callback.ts

```diff
@@ -17,10 +17,13 @@ export async function onModelTextUpdated(
   await richText.updateComplete;
   const vEditor = richText.vEditor;
   assertExists(vEditor, 'VEditor is not ready yet.');
-  vEditor.slots.updated.once(() => {
-    if (callback) {
-      callback(richText);
-    }
+  await new Promise<void>(resolve => {
```

Comment:
This seems to be simply a `queueMicrotask` alternative. But semantically here shall we wait for an extra tick here? This may lose emitted events and potentially break other behavior.

Reply:
Yes, I'll look into it further.